README.md

```diff
@@ -782,13 +516,6 @@ with DOM nodes][morphdom-bench], and it has the added benefit of working with
 _any_ library that produces valid DOM nodes. So to put a long answer short:
 we're using something even better.
 
-### What packages do you recommend to pair with choo?
-- [tachyons](https://github.com/tachyons-css/tachyons) - functional CSS for
-  humans
-- [sheetify](https://github.com/stackcss/sheetify) - modular CSS bundler for
-  browserify
-- [pull-stream](https://github.com/pull-stream/pull-stream) - minimal streams
-
 ### How can I optimize choo?
```

Comment:
We should mention that es2020 should be run as a global transform here or they might not see it later on in the testing section (and since the `unassertify` transform is mentioned in that state as well).

also in the handbook or in an example we might want to show how to set up browserify to be opinionated in the package.json so something consuming your component might not need to leak out any of these details?